ipa-core/benches/oneshot/ipa.rs

```diff
@@ -61,6 +61,12 @@ struct Args {
     /// while doing modulus conversion and attribution
     #[arg(long, default_value = "3")]
     num_multi_bits: u32,
+    /// DP parameters. Will run with DP by default. Can only be run without DP if `with_dp` == 0.
+    /// in which case the value of `epsilon` is ignored.
+    #[arg(short = 'd', long, default_value = "1")]
+    with_dp: u32,
```

Comment:
have you tried to make it work with `Option`? I remember you mentioning some issues there, wondering if that what prevented you from doing it here

Reply:
I tried a few things. I'm not sure `Option` on only epsilon was one of them or not.  But it is working not so don't want to mess with it anymore for now.